integrations/ultralytics-yolov5/tutorials/sparsify-from-scratch.md

```diff
@@ -0,0 +1,409 @@
+<!--
+Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Sparsifying YOLOv5 From Scratch
+
+This page explains how to sparsify a YOLOv5 model from scratch with SparseML's CLI.
+
+## Overview
+
+Sparsifying a model involves removing redundant information from a 
+trained model using algorithms such as pruning and quantization. The sparse models can then be deployed with DeepSparse, which implements many optimizations to increase performance via sparsity, for GPU-class performance on CPUs.
+
+In this tutorial, we will demonstrate how to use recipes to create 
+sparse versions of YOLOv5.
+
+**Pro Tip**: For YOLOv5, there are pre-sparsified checkpoints of each version available in [SparseZoo](https://sparsezoo.neuralmagic.com/?domain=cv&sub_domain=detection&page=1). 
```

Comment:
Pro Tip: For YOLOv5, each version has pre-sparsified checkpoints available in SparseZoo.